Vagrantfile

```diff
@@ -165,26 +151,21 @@ Vagrant.configure(2) do |config|
     ${HOME}/bin/direnv allow ${HOME}/scf
   SHELL
 
-  config.vm.provision "shell", privileged: false, inline: <<-SHELL
-    set -e
-
+  # Install common and dev tools
+  config.vm.provision :shell, privileged: true, inline: <<-SHELL
+    set -o errexit -o xtrace -v
     # Get proxy configuration here
-    source /etc/environment
-    export no_proxy http_proxy https_proxy NO_PROXY HTTP_PROXY HTTPS_PROXY
-
-    # Install development tools
-    (
-      cd "${HOME}/scf"
-      ${HOME}/bin/direnv exec ${HOME}/scf/bin/dev/install_tools.sh
-    )
-
+    export HOME=/home/vagrant
+    export PATH=$PATH:/home/vagrant/bin
+    cd "${HOME}/scf"
+    bash ${HOME}/scf/bin/common/install_tools.sh
```

Comment:
I thought we ended up conceding (maybe a bit begrudgingly) that `/usr/local/bin` was a good place to put executables, because it's not tied to any user that might run them.

The helm owner was `vagrant:vagrant` when extracted by root, because 1000:1000 is apparently the permission in the helm tgz.. to avoid that, I passed the `--no-same-owner` flag to the tar extraction command.

The tools directory is gone now.. as far as I could tell, we weren't using it for anything.. I put the cf.tgz used for extracting the cf binary into /tmp instead